Documentation/spelling_wordlist.txt

```diff
@@ -908,6 +910,7 @@ xfrm
 xml
 xor
 xoring
+xt
```

Comment:
what is `xt`?

Reply:
I think it comes from `xt_socket`, it looks like Sphinx splits words on underscores :(.

We could maybe change that behaviour by implementing a [dedicated filter](https://sphinxcontrib-spelling.readthedocs.io/en/latest/customize.html?#custom-word-filters) for the spelling plugin :thinking:

Reply:
I agree — but maybe as a subsequent PR that takes care of all these cases (and eventually of addressing the non-goals)?

Reply:
@qmonnet Thanks for clarifying!

Reply:
@bmcustodio I don't know if it's worth doing at all, I was mostly thinking out loud. But yeah, definitely for another PR if we decided to tackle this.